1-reverse-number/abekroenem/reverse_number.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+# Resolução do desafio
+def reverse(number:int) -> int:
+    return int(str(number)[::-1])
+
+# Teste unitário
+class ReverseTestCase(unittest.TestCase):
```

Comment:
Em python é super comum escrever bastante class nas coisas mas dá pra não usar e deixar o código um pouco mais funcional.

http://ginstrom.com/scribbles/2008/10/06/dont-overuse-classes-in-python/

Reply:
@anabastos agradeço pela revisão e pelo post.